api/registrations/views.py

```diff
@@ -268,33 +275,17 @@ def get_queryset(self):
         return queryset
 
 
-class RegistrationChildrenList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin, RegistrationMixin):
+class RegistrationChildrenList(BaseChildrenList, generics.ListAPIView, ListFilterMixin, RegistrationMixin):
     """The documentation for this endpoint can be found [here](https://developer.osf.io/#operation/registrations_children_list).
     """
     view_category = 'registrations'
     view_name = 'registration-children'
     serializer_class = RegistrationSerializer
 
-    permission_classes = (
-        ContributorOrPublic,
-        drf_permissions.IsAuthenticatedOrReadOnly,
-        ReadOnlyIfRegistration,
-        base_permissions.TokenHasScope,
-        ExcludeWithdrawals,
-    )
-
     required_read_scopes = [CoreScopes.NODE_REGISTRATIONS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    ordering = ('-modified',)
-
-    def get_default_queryset(self):
-        return default_node_list_permission_queryset(user=self.request.user, model_cls=Registration)
```

Comment:
This line is what caused me to go down a rabbit hole of modifying `default_node_list_permisison_queryset` (this PR has been closed) - NodeChildrenList was not using this, but RegistrationChildrenList was.   Since this queryset was further being filtered to just children of the current registration, and then from there, filtering on ones the user had permission to view, the return of `get_default_queryset` needed to include the nodes you could see with VOL's (otherwise, by the time you got around to the last step of checking permissions, the queryset wouldn't have VOL nodes in it to begin with).

Instead changing so `default_node_list_queryset` is used, which just returns non-deleted nodes, to match NodeChildrenList, and then contributor permissions/VOL's/implicit admin perms can be taken care of in Registration.objects.can_view()